### PR TITLE
[optin/puller] task: workflow/populate db

### PR DESCRIPTION
**Pull request by gimre-xymcity**

### CODE REVIEW

File: optin/puller/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## optin/puller order of workflows
+
+
+* workflows/popuplate_db
```
Comment (Jaguar0625) [inline optin/puller/README.md:4]: spelling

File: optin/puller/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## optin/puller order of workflows
+
+
```
Comment (Jaguar0625) [inline optin/puller/README.md:3]: one blank line ?

File: optin/puller/puller/db/CompletedOptinDatabase.py [hunk 3]
```diff
@@ -69,3 +74,5 @@ def insert_mappings_from_json(self, mappings_json):
 			}, {
 				destination_json['sym-address']: destination_json['sym-balance'] for destination_json in mapping_json['destination']
 			})
+
+			print('.', end='', flush=True)
```
Comment (Jaguar0625) [inline optin/puller/puller/db/CompletedOptinDatabase.py:78]: ?

File: optin/puller/puller/db/CompletedOptinDatabase.py [hunk 4]
```diff
@@ -39,7 +39,12 @@ def insert_mapping(self, nem_address_dict, symbol_address_dict):
 		nem_balance = reduce(lambda x, y: x + y, nem_address_dict.values())
 		symbol_balance = reduce(lambda x, y: x + y, symbol_address_dict.values())
 
-		if nem_balance != symbol_balance:
+		ngl_source = [
+			'NATRUSTUAB5LAWDSOWDUPQUFYRTGZTJZDGML2JKP',
+			'NANODESTSN7GU76QPLGMJ7BCGCAA2PHVBZZUUI62',
+			'NBKXLNQ2GKTLQBTXNFOHULNJDA7T2Q57CZGQ2TFP'
+		]
+		if nem_balance != symbol_balance and set(ngl_source) != set(nem_address_dict.keys()):
```
Comment (Jaguar0625) [inline optin/puller/puller/db/CompletedOptinDatabase.py:47]: can we just modify json instead of this hack ? 😅

Comment (gimre-xymcity) [reply optin/puller/puller/db/CompletedOptinDatabase.py:47]: not sure, but I think it's rather make more sense to have it visible in the code, could additionally throw in amounts if you want

Comment (Jaguar0625) [reply optin/puller/puller/db/CompletedOptinDatabase.py:47]: i'd rather not special case this 😅

**Comment by Jaguar0625** [comment]
fix lint ^^

**Pull request closed without merging**